### ISSUE TITLE
Nothing generated after install the operator and install replica example

### ISSUE THREAD

**Issue by victory4software** [mongodb/mongodb-kubernetes-operator#1035]
**What did you do to encounter the bug?**
1. I installed the operator using 
helm -n mongo-operator install community-operator mongodb/community-operator
 
![image](https://user-images.githubusercontent.com/28552955/176830564-0fa91ce1-0a39-49ed-af31-6e26d34048e8.png)

2.I tried to install and example of yaml file
```
---
apiVersion: mongodbcommunity.mongodb.com/v1
kind: MongoDBCommunity
metadata:
  name: example-mongodb
spec:
  members: 3
  type: ReplicaSet
  version: "4.2.6"
  security:
    authentication:
      modes: ["SCRAM"]
  users:
    - name: root
      db: admin
      passwordSecretRef: # a reference to the secret that will be used to generate the user's password
        name: my-user-password
      roles:
        - name: clusterAdmin
          db: admin
        - name: userAdminAnyDatabase
          db: admin
      scramCredentialsSecretName: my-scram
  additionalMongodConfig:
    storage.wiredTiger.engineConfig.journalCompressor: zlib

# the user credentials will be generated from this secret
# once the credentials are generated, this secret is no longer required
---
apiVersion: v1
kind: Secret
metadata:
  name: my-user-password
type: Opaque
stringData:
  password: xxxxxxxxx
```
~                                              
after apply the changes of tha above file i a different namespace nothing generated on this except the custom resources
![image](https://user-images.githubusercontent.com/28552955/176830929-94be956c-d6e4-4542-acfc-e1ef4f51f337.png)

**What did you expect?**
I am expecting the secret that contains the connection string details to be generated but nothing is generated

**What happened instead?**
No pods or secretes generated after apply the above yaml code

**Operator Information**
 - Operator Version community-operator-0.7.4
 -
 - MongoDB Image used
type: ReplicaSet
  version: "4.2.6"

**Kubernetes Cluster Information**
 Minikube

**Comment by nayrnet** [comment]
I have also encountered this, after ours failed we built a clean sandbox cluster and followed the instructions step by step, it did not result in an example-mongodb running.. cannot tell where the failure is happening at, nothing in the operator logs and the CRD resource has no status or phase.

**Comment by slaskawi** [comment]
I've just followed the [Helm-based installation guide](https://github.com/mongodb/mongodb-kubernetes-operator/blob/master/docs/install-upgrade.md#install-the-operator-using-Helm) and everything seems to be working fine:

```
$ helm install community-operator mongodb/community-operator --namespace mongodb-dfadfa13418 --create-namespace
NAME: community-operator
LAST DEPLOYED: Tue Aug 23 15:08:55 2022
NAMESPACE: mongodb-dfadfa13418
STATUS: deployed
REVISION: 1
TEST SUITE: None

$ kubectl apply -f ./config/samples/mongodb.com_v1_mongodbcommunity_cr.yaml 
mongodbcommunity.mongodbcommunity.mongodb.com/example-mongodb created
secret/my-user-password created

$ kubectl get pods   
NAME                                          READY   STATUS    RESTARTS   AGE
example-mongodb-0                             2/2     Running   0          8m23s
example-mongodb-1                             2/2     Running   0          6m57s
example-mongodb-2                             2/2     Running   0          6m11s
mongodb-kubernetes-operator-c486d9f7d-djjhl   1/1     Running   0          12m
```

Closing it for now. Please reopen if there's anything else I can help you with.

**Comment by slaskawi** [comment]
After re-reading the issue, I think there's one thing that's worth highlighting. The `MongoDBCommunity` you pasted doesn't have a namespace. This means it will be created in the namespace that is currently set in your `kubectl` context. I highly recommend running `kubectl config set-context --current --namespace=<your namespace, the same that you set in --namespace switch of helm command>`

**Comment by dan-mckean** [comment]
Hi, I'm Dan and I'm the Product Manager for MongoDB's support of Kubernetes.

I'm doing some work right now to try and identify how the Community Operator is being used. The Community Operator is something I inherited when I started at MongoDB, but it doesn't get as much attention from us as we'd like and we're trying to understand how it's used in order to establish its future. It will help us prioritize future issues and PRs raised by the community 🙂

Here's a super short survey (it's much easier for us to review all the feedback that way!): [https://docs.google.com/forms/d/e/1FAIpQLSfwrwyxBSlUyJ6AmC-eYlgW_3JEdfA48SB2i5--_WpiynMW2w/viewform?usp=sf_link](https://docs.google.com/forms/d/e/1FAIpQLSfwrwyxBSlUyJ6AmC-eYlgW_3JEdfA48SB2i5--_WpiynMW2w/viewform?usp=sf_link)

 If you'd rather email me instead: [dan.mckean@mongodb.com](mailto:dan.mckean@mongodb.com?subject=MongoDB%20Community%20Operator%20feedback)

Thank you in advance!
Dan
